skops/hub_utils/_hf_hub.py

```diff
@@ -73,6 +74,34 @@ def _validate_folder(path: Union[str, Path]) -> None:
         raise TypeError(f"Model file {model_path} does not exist.")
 
 
+def _convert_to_2d_numpy_array(data):
```

Comment:
we should use `sklearn.utils.validation.check_array` instead of writing our own, unless we handle more input types.